catalog/spatial/registry/registry-rest-endpoint/src/main/resources/templates/organizations.hbt

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta content="text/html; charset=windows-1252" http-equiv="content-type"/>
```

Comment:
`windows-1252` was originally in the report handlebars before, I have changed it to `UTF-8` though